### DISCUSSION TITLE
Feature Requests : support echarts  in nicegui?

### DISCUSSION THREAD

**Discussion by nicedouble** [zauberzeug/nicegui#1324] [Ideas / Feature Requests]
nicegui implements interactive plotting through Highcharts and Plotly.
Is it possible to support [echarts](https://echarts.apache.org/en/index.html) in the future?

**Comment by falkoschindler** [comment]
Yes, @nicedouble, that would certainly be possible. The API looks similar to Highcharts with a large options dictionary, so the integration should be similar as well.
Anyone up for a try? A pull request would be very welcome. 🙂

**Comment by natankeddem** [comment]
I could try tackling this after I get Highcharts draggable points going since that is getting me pretty familiar with how the Highcharts element is working.

**Reply by rodja** [reply]
Super cool! I'm looking forward to your pull requests.

**Reply by natankeddem** [reply]
It isn't perfect but it's a start...
https://github.com/zauberzeug/nicegui/pull/1346

**Comment by falkoschindler** [comment]
I'll close this feature request as implemented in #1346.
